src/release/breaking-changes/ios-flutterviewcontroller-splashscreenview-nullable.md

```diff
@@ -0,0 +1,100 @@
+---
+title: iOS FlutterViewController splashScreenView make nullable
+description: FlutterViewController splashScreenView changed from nonnull to nullable.
+---
+
+## Summary
+
+We changed property `splashScreenView` of `FlutterViewController` from `nonnull` to `nullable`.
+
+Old declaration of `splashScreenView`:
+
+```objective-c
+@property(strong, nonatomic) UIView* splashScreenView;
+```
+
+New declaration of `splashScreenView`:
+
+```objective-c
+@property(strong, nonatomic, nullable) UIView* splashScreenView;
+```
+
+## Context
+
+This allows iOS add-to-app users writen in `Swift` to remove the splash screen view.
+
+However, the previous declaration would build error when set `nil` value using `Swift`, only supported if using `Objective-C`.
+
+## Description of change
+
+Before migrated, it would be a compilation error if the calling code is in Swift and gets the splash screen and stores in a nonnull UIView variable.
+
+```
+error build: Value of optional type 'UIView?' must be unwrapped to a value of type 'UIView'
+```
```

Comment:
````suggestion
While it was possible in Objective-C to work around the incorrect `nonnull` annotation by setting
`splashScreenView` to a `nil` `UIView`, in Swift this caused a compilation error:
```
error build: Value of optional type 'UIView?' must be unwrapped to a value of type 'UIView'
```

[PR #34743][] updates the property attribute to `nullable`. It can return `nil` and can be set to `nil` to remove the view in both Objective-C and Swift.
````